lib/dav4rack/xml_response.rb

```diff
@@ -71,35 +68,28 @@ def render_lockdiscovery(*args)
 
 
     # returns an activelock Ox::Element for the given lock data
-    def activelock(time: nil, token:, depth:,
-                      scope: nil, type: nil, owner: nil, root: nil)
-
+    def activelock(options)
```

Comment:
I would really prefer to not go back from keyword arguments to an options hash.

Reply:
Okay, reverted.